src/Microsoft.Identity.Web.OWIN/HttpContextBaseExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Web;
+using Microsoft.Graph;
+
+namespace Microsoft.Identity.Web
```

Comment:
We'll change this to extension methods on the Controller or ControllerBase.

Reply:
ok. will update.